### ISSUE TITLE
support - machine powers off after a short time

### ISSUE THREAD

**Issue by rtpHarry** [Varying-Vagrant-Vagrants/VVV#1610]
My VVV setup has crumbled. It keeps powering off and I'm not sure where to look for logs like this that will point to a smoking gun.

I've upgraded everything now as you can see below. I've even started a new box and just loaded it up with one site and left it just sitting there and then when I come back and try to refresh the phpmyadmin it times out and I have to `vagrant up` again.

I've tried to put the ram back down to 1024 even though I have 16gb in the machine I thought it was worth trying.

I've rebooted the host machine multiple times.

When I load up the Oracle VM Virtualbox Manager the machines are always showing as Powered Off since I upgraded to VVV 2.2.1 (and the underlying Vagrant stuff as outlined in the blog post).

Before when I started experiencing the issue with 2.0.0 it was showing as powered on until it crashed then showed powered off.

If I try to go to the properties of the box in Oracle VM Virtualbox Manager, it's in powered off state and I get this:

![image](https://user-images.githubusercontent.com/1038062/44132790-03d6e430-a086-11e8-8e84-240d9bffb4ab.png)

I've lost 4 days now and I'm starting to get desperate as its really difficult to do my job without this. I'm willing to nuke everything if there is something I can do to clean it out and get it back working. Or some log that can show. 

I have seen a setting somewhere that I think is something like "auto shut down on error" but I don't know where the log for that error is going to or how I can move forward.

I've also done the `vagrant box update` command and its on the latest release from a few days ago.

## Your Environment

* VVV version: 2.2.1
* VVV Git Branch: master
* Vagrant version: 2.1.2
* VM Provider name: VirtualBox
* VM Provider version: 5.2.16, updated guest additions as well
* Operating System and version: Windows 10

**Comment by tomjn** [comment]
So Vagrant boxes automatically turn off when the battery on a machine runs low so that the VM is safely shut down, but what you're reporting is unexpected.

I would normally suggest switching to the `develop` branch as it's usually got bugfixes and is quite stable, but in this looks like more of a virtualbox issue.

When you say you started a new box, was that in the same folder? Or was it in a new folder as a new instance?

I would consider the following options:

 - `vagrant destroy` and reprovision
 - Try and uninstall, reboot, then reinstall VirtualBox

Judging from the error message in the screenshot I don't think VVV or Vagrant are likely causes. In the past I haven't seen similar issues, but Windows updates can sometimes give VirtualBox pause for thought.

Let me know how a clean reinstall of VirtualBox goes, make sure to check for backups of the database first though. If VirtualBox has logs they may be helpful in determining why the box shuts down early

**Comment by rtpHarry** [comment]
Thanks for the detailed reply. I'm going to try a full uninstall of VirtualBox.

The laptop is plugged in so isnt a battery issue. I tried in the same folder initially then moved over to new folder. Tried to destroy and rebuild.

Just tried to use the develop branch and got this error 

```
The guest machine entered an invalid state while waiting for it
to boot. Valid states are 'starting, running'. The machine is in the
'unknown' state. Please verify everything is configured
properly and try again.

If the provider you're using has a GUI that comes with it,
it is often helpful to open that and watch the machine, since the
GUI often has more helpful error messages than Vagrant can retrieve.
For example, if you're using VirtualBox, run `vagrant up` while the
VirtualBox GUI is open.

The primary issue for this error is that the provider you're using
is not properly configured. This is very rarely a Vagrant issue.
```

So something isn't right.

**Comment by rtpHarry** [comment]
I've fully uninstalled, gone through all the folders and deleted all config files and totally started from fresh but its still only lasting 15 minutes before the box turns itself off and I cannot find any log files that show why its crashing.

I've noticed in the changelog that it says this:

```
## 2.2.1 (May, 2018)

Note that to update to 2.2.1, you must remove the Vagrant triggers plugin and install Vagrant 2.1
```

Is this a typo or was I supposed to upgrade from 2.0.0 to 2.1.0 and then 2.2.1?

**Comment by tomjn** [comment]
If you're on 2.2+ it should not matter how you arrived there

Did you check the logs for VVV itself?

**Comment by tomjn** [comment]
To clarify, if you are on the correct version of vagrant and the latest
VVV, then how you got there is irrelevant

**Comment by tomjn** [comment]
> Is this a typo or was I supposed to upgrade from 2.0.0 to 2.1.0 and then 2.2.1?

Re-reading this, no it isn't a typing **VVV 2.2.1** vs _Vagrant 2.1_

**Comment by rtpHarry** [comment]
Well after a rough week of watching shell scripts slowly tick by on my dodgy Vietnamese internet connection I finally gave up. I checked the release dates for when I got this laptop (last Sept) and fished out the releases of Vagrant and VirtualBox from then.

I'm running Vagrant 1.9.8, VirtualBox 5.1.26 and VVV 2.1.0 and it hasn't crashed for a while now. I've managed to test it out with a fresh box and then recover my original dev box.

I wrote a only-works-on-my-machine backup script which I use to boot up vagrant and it backs up the database backup folder, called [devup](https://bitbucket.org/rtpharry/devup/). It saved me again because the database had all got reset during the madness so I managed to pull them out of my archives and put it back on its feet.

So I guess I'm trapped in amber until I get desperate enough to try upgrading again :(

Seems its an issue with Vagrant or VirtualBox, perhaps on Windows 10. Whatever it is I hope it goes wrong for other smarter users then myself and they manage to diagnose the bug and solve it before my next attempt.

Guess that's the end of my contributions for the moment as well... thanks for the detailed help.
